src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    list-style-type: none;
+  }
+
+  &__item:not(:last-child) {
+    padding-right: 20px;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $color-light-black;
+    display: inline-block;
+    padding-top: 23px;
+    padding-bottom: 22px;
+    font-weight: $medium-font-weight;
+    font-size: $small-font-size;
+    text-transform: uppercase;
+  }
+
+  .is-active {
+    color: $color-accent-blue;
+    padding-bottom: 0;
+  }
+
+  .is-active::after {
```

Comment:
this element should be positioned at the bottom
add `position: relative, bottom: 0, left: 0`